ProfileAPI/ProfileAPI/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using ProfileAPI.Models;
+using ProfileAPI.Services;
+
+namespace ProfileAPI.Controllers
```

Comment:
Could we change the namespace to ProfileService to be consistent with the ConferenceService? If not, it should pascal case: ProfileApi

Reply:
I don't like having the same name for the project and service